docs/en/sql-reference/functions/date-time-functions.md

```diff
@@ -435,6 +435,166 @@ Result:
 
 -   [toStartOfInterval](#tostartofintervaltime-or-data-interval-x-unit-time-zone)
 
+## date\_add {#date_add}
```

Comment:
But how do these functions work?
What will happen if we will add one month to 31th January?

Reply:
I think these functions were written to make them compatible with MySQL functions. For example https://www.w3schools.com/sql/func_mysql_date_add.asp. 
So I assume that they are expected to work how they work with MySQL ?
To answer your question adding 1 month to Jan 31st:
```
select DATE_ADD(MONTH, 1, toDateTime('2021-01-31'));

┌─plus(toDateTime('2021-01-31'), toIntervalMonth(1))─┐
│                                2021-02-28 00:00:00 │
└────────────────────────────────────────────────────┘
```

Reply:
https://www.w3schools.com/sql/func_mysql_date_add.asp
The type of interval to add. Can be one of the following      values:            MICROSECOND  

probably we can use the same sentence `The type of interval `

Reply:
Sure, reworded the def of `unit` to use `The type of interval ....` wherever relevant.